src/components/common/Table.js

```diff
@@ -56,13 +56,18 @@ export default function Table({
   useEffect(() => {
     // Reset to first page when filters are changed
     table.gotoPage(0);
-  }, [filterState, forceResetPage]);
+    // eslint-disable-next-line no-console
```

Comment:
Think you forgot to remove this 👍

Reply:
Oops thanks missed that and an unused prop